libtsuba/include/katana/TxnContext.h

```diff
@@ -100,6 +109,8 @@ class KATANA_EXPORT TxnContext {
   bool all_properties_write_{false};
   bool topology_read_{false};
   bool topology_write_{false};
+
+  bool commit_manifest_{true};
```

Comment:
What does this Boolean mean?  It seems like `Commit `or `CommitToStorage `would be a function on TxnContext that would commit in-memory state, like a manifest file.  But this could be an `early_commit `flag to write the manifest as soon as we know its name?

Reply:
Do you mean you want `Commit` and `CommitToStorage` to be member functions of `TxnContext`? Right now they are member functions of `DistPropertyGraph`. Graphs are not cached in `TxnContext`. The boolean `commit_manifest_` is used to indicate a function call with a `TxnContext` passed in as parameter whether to commit the manifest to storage. For transaction related operations, such as transactions and single-operations as transactions, they define a `TxnContext` with this boolean set to be false. So that `libtsuba` does not commit the manifest to storage. Otherwise, `commit_manifest_` is by default true, meaning that all tests and tools not related to operations in the server/graph-worker context will commit the manifest to storage in `libtsuba`. 

In this sense, we can remove the setter function for this boolean, and only allow it to be defined in `TxnContext` constructors. Do you want me to remove the setter or just remain it?